sdk/data-generators/image-processing-generator/Makefile

```diff
@@ -10,6 +10,9 @@
 # and copyright information.
 
 
+# "Puerto de Veracruz" by Eduardo Pavon is licensed under CC BY-SA 2.0
```

Comment:
I think the `LICENSE.markdown` file needs updating with the text of the license, and a notice that we are using that image.

Reply:
Should I append CC-BY-SA 2.0 to `LICENSE.markdown`?

Reply:
I think so, yes.  Probably add each license in a separate section, with the section name being the SPDX identifier of the license, or whatever.  Also, add a short preamble indicating which files are licensed under the `CC-BY-SA 2.0` license (and pointing out that the Veracruz codebase itself is licensed under the `MIT` license).

Reply:
Done. I followed this example: https://github.com/github/choosealicense.com#license
The README specifies what is covered by each license. `LICENSE.markdown` is left untouched.